content/breach.md

```diff
@@ -52,11 +50,14 @@ There are no Ethereum transactions necessary for this breach.
 
 ## I’m self-hosted—what do I need to do?
 
-Simply use the `flagday` tool that will be provided. It will perform all the relevant actions on your behalf, as well as exporting and reimporting your data.
+**Simply use the `flagday` tool that we have provided**. It will perform all the relevant actions on your behalf, as well as exporting and reimporting your data. You can find the `flagday` script [here](https://gist.github.com/philipcmonk/8e3d095b9545069237c759cd9aad32c2). Usage instructions are provided in the comments at the top of the file. It's safe to use `flagday` now, but **it's not necessary to do so until after the breach.**
+
+Alternatively, you can perform the migration manually (see below).
+
+## Can I perform the migration manually?
 
-`flagday` will become available for use on or before ~2020.12.4.
+Yes—follow these steps:
```

Comment:
Should mention that these manual instructions *do not* migrate your user data (group subscriptions, backlog, etc).